src/infill.cpp

```diff
@@ -4,6 +4,10 @@
 #include "utils/polygonUtils.h"
 #include "utils/logoutput.h"
 
+#define SQRT2MUL(x) ((30547*(x))/21600)
+#define OCTSLEN(x) ((43200*(x))/73747)
```

Comment:
Yes. Inclusive.
Or it's `1/sqrt(3)` which is the ratio between the side and the diagonal of a cube, but with severe rounding errors :P